src/app/code/community/Omise/Gateway/controllers/Callback/ValidateoffsitetescoController.php

```diff
@@ -1,7 +1,7 @@
 <?php
 class Omise_Gateway_Callback_ValidateoffsitetescoController extends Omise_Gateway_Controller_Base
 {
-    const PAYMENT_TITLE = 'True Money';
+    const PAYMENT_TITLE = 'Bill Payment: Tesco';
```

Comment:
Maybe 'Tesco Bill Payment'? It appears differently in different places. Similar to above comment